src/components/POLine/Location/LocationForm.js

```diff
@@ -71,7 +76,12 @@ const LocationForm = ({ parentResources }) => (
             name={`${field}.quantityPhysical`}
             parse={parseQuantity}
             type="number"
-            validate={validateQuantityPhysical}
+            validate={[
+              validateQuantityPhysical,
+              ([...PHRESOURCES, OTHER].includes(orderFormat)
+                ? validateRequiredNotNegative
+                : noop)]}
+            required={[...PHRESOURCES, OTHER].includes(orderFormat)}
```

Comment:
`[...PHRESOURCES, OTHER].includes(orderFormat)` is executed twice

Reply:
Thanks, good point. Fixed